src/gripper_controller/gripper_controller.py

```diff
@@ -20,30 +20,20 @@ def __init__(self):
         self.success = True
 
         # server
-        self.srv_safe_joint_change_left = actionlib.SimpleActionServer('left_arm/gripper/action',
-                                                                       GripperCommandAction,
-                                                                       execute_cb=self.gripper_left,
-                                                                       auto_start=False)
-        self.srv_safe_joint_change_right = actionlib.SimpleActionServer('right_arm/gripper/action',
-                                                                        GripperCommandAction,
-                                                                        execute_cb=self.gripper_right,
-                                                                        auto_start=False)
-        self.srv_safe_joint_change_left.start()
-        self.srv_safe_joint_change_right.start()
+        self.srv_safe_joint_change = actionlib.SimpleActionServer('/hero/gripper/action',
```

Comment:
Why this hardcoded `hero` prefix?

Reply:
Because this entire unit is hero-specific. No need to pretend that it is any more generic than it is

Reply:
But his is fixed by namespacing the node. Keep it consistent.
```suggestion
        self.srv_safe_joint_change = actionlib.SimpleActionServer('gripper/action',
```